src/core/sceneentity.h

```diff
@@ -135,6 +135,8 @@ class KUESASHARED_EXPORT SceneEntity : public Qt3DCore::QEntity
     Qt3DRender::QTextureLoader *m_brdfLUT;
 };
 
+SceneEntity *findParentSceneEntity(Qt3DCore::QEntity *entity);
```

Comment:
shouldn't this be in `kuesa_utils_p.h`?
And if it stays here, shouldn't it be a static public member of SceneEdity?
Or if it stays a separate function, shouldn't it have `KUESASHARED_EXPORT`?

Reply:
Changed. Now is a static method of SceneEntity.